### DISCUSSION TITLE
Forgetabble tale: Quest helper with sidebar open generates strong input lag during quest.

### DISCUSSION THREAD

**Discussion by Swent-SW** [Zoinkwiz/quest-helper#1733] [Q&A]
As said in title. It only happened during this quest and with the sidebar open. If I close the sidebar it is immediately solved. Changing to a different plugin also solves the issue

**Comment by pajlada** [comment]
We've had a second report of the same issue in the Discord, but me or @Haavardaw are unable to reproduce it. currently looking more into it

**Reply by Swent-SW** [reply]
It only occurs during the part of the quest before the puzzles. Especially when collecting the 4 seeds and brewing the beer. I found there was a similar bug with the farming timer, so maybe it is related to this?

**Reply by pajlada** [reply]
Seems like it's related to https://github.com/Zoinkwiz/quest-helper/pull/1655 - will have a look this weekend to see if I can figure it out

**Reply by Swent-SW** [reply]
Just to expand on this in case it might help you (because the plugin still works with the ribbon closed, so it is very useable). I started While Guthix Sleeps and encountered the same issue. Uninstalling and Reinstalling the plugin does not seem to fix it. It might be a user thing? Would love to provide any information that might help you

**Reply by pajlada** [reply]
I can't reproduce it, nor can 5 other people I've asked to test.
You and 2 more people in Discord have experienced it though, and from my testing with one of them, it wasn't specific to a quest.
We did some bisecting and found it was the above-linked PR that started the input lag. I'm currently playing around with a build & having them test it, but it's not trivial to test unless you have a RuneLite development setup on already.

**Reply by pajlada** [reply]
Looks like https://github.com/Zoinkwiz/quest-helper/pull/1734 will fix it, if you have a Java developer setup feel free to test it

**Reply by Swent-SW** [reply]
I'm afraid I don't have a Java Developer setup :/ I just wanted to help flag something as an end-user. Not much of a developer myself atm

**Reply by pajlada** [reply]
No worries, I appreciate the report either way. Helps to know that it's not just one person in Discord having the issue
